docs/user/dashboard/tsvb.asciidoc

```diff
@@ -132,6 +132,26 @@ To change this behavior, click *Panel options*, then specify a URL in the *Item
 
 The *Markdown* visualization supports Markdown with Handlebar (mustache) syntax to insert dynamic data, and supports custom CSS.
 
+[float]
+[[edit-visualizations-in-lens]]
+==== Edit visualizations in Lens
+
+Open and edit your *TSVB* visualizations in *Lens*, which is the drag-and-drop visualization editor that provides you with more visualization types, reference lines, and more.
+
+To get started, click *Edit visualization in Lens* in the toolbar.
```

Comment:
This looks great! Do we want to mention that this work only for the timeseries TSVB charts atm?